### ISSUE TITLE
Opening keyboard shortcuts PDF doesn't work

### ISSUE THREAD

**Issue by ubruhin** [LibrePCB/librepcb-snap#5]
When opening the keyboard shortcuts from LibrePCB, nothing happens and this is printed to stderr:

```
[  INFO   ] Successfully opened URL with /bin/xdg-open: "file:///tmp/librepcb/librepcb-shortcuts-reference.pdf"
user-open error: request declined by the user (code 2)
```

Looks exactly like the problem described here: https://forum.snapcraft.io/t/please-allow-classic-confinement-for-gmailctl/24233

**Comment by dbrgn** [comment]
I'm pretty sure this is because the file was written outside the sandbox, but is opened from within the sandbox, or vice versa.

**Comment by ubruhin** [comment]
Fixed in bc8fc5d0dc011fb29834170bb1d725407eed1936
